docs/reference-architectures/powerbi.rst

```diff
@@ -0,0 +1,182 @@
+    | node                                   | uri                                       | success_count | error_count | errors |
+    +----------------------------------------+-------------------------------------------+---------------+-------------+--------+
+    | {"id": "...", "name": "Hoher Riffler"} | file:///path/to/population_data0_.json.gz |        310991 |           0 | {}     |
+    | {"id": "...", "name": "Hoher Riffler"} | file:///path/to/population_data1_.json.gz |        309975 |           0 | {}     |
+    | {"id": "...", "name": "Hoher Riffler"} | file:///path/to/population_data2_.json.gz |        309586 |           0 | {}     |
+    | {"id": "...", "name": "Hoher Riffler"} | file:///path/to/population_data3_.json.gz |        309980 |           0 | {}     |
+    +----------------------------------------+-------------------------------------------+---------------+-------------+--------+
+
+
+Creating a DSN for CrateDB
+--------------------------
+
+First, verify that you have installed the PostgreSQL ODBC driver by searching
+for and opening the ODBC Data Sources application for your architecture:
+
+.. figure:: powerbi-odbc-start-menu.png
+   :align: center
+
+Navigate to the drivers tab, where you should see the PostgreSQL ODBC drivers
+you have installed:
+
+.. figure:: powerbi-odbc-postgres-drivers.png
+   :align: center
+
+If this is correct, you can add a User DSN by navigating to the User DSN tab
+and hitting 'Add'. From the drivers, select *PostgreSQL Unicode(x64)* if you
+are using a 64-bit Windows. Otherwise, select *PostgreSQL Unicode(x32)*.
+
+.. figure:: powerbi-driver-select.png
+   :align: center
+
+Set up the data source with the following parameters:
+
++---------------+--------------------------------------------+
+| **Parameter** | **Value**                                  |
++===============+============================================+
+| Data Source   | ``CrateDB``                                |
++---------------+--------------------------------------------+
+| Server        | The URL or IP of your CrateDB server.      |
++---------------+--------------------------------------------+
+| User Name     | The CrateDB user you wish to connect with. |
++---------------+--------------------------------------------+
+| Port          | ``5432``                                   |
++---------------+--------------------------------------------+
+
+If your CrateDB instance uses SSL, enable the SSL mode. If the CrateDB user
+you're connecting with is password protected, add its password to the Password
+field.
+
+.. figure:: powerbi-dsn-setup.png
+   :align: center
+
+You can hit *Test* to verify that the DSN is configured correctly. Once this
+is done, hit *Save* to save the DSN.
+
+
+Connecting Power BI to CrateDB
+------------------------------
+
+Once you have created the DSN, open up Power BI and click 'Get Data' from the
```

Comment:
You should be consistent about how you represent things to click and interact with: either all 'like so' or all _like so_

Reply:
Which do you think works best? I think i'd prefer *like this* but I don't know if we have a standard.

Reply:
Yeah in the Cloud docs I've generally used italics for that